packages/core/lib/commands/db/meta.js

```diff
@@ -22,14 +22,8 @@ module.exports = {
   },
 
   subCommands: {
-    serve: {
-      help: serveCommand.help,
-      description: serveCommand.meta
-    },
-    query: {
-      help: queryCommand.help,
-      description: queryCommand.meta
-    }
+    serve: serveCommand.meta,
```

Comment:
Isn't the idea that the structure for these should mirror that of the main commands or am I missing something? Like, for the db command there is both a `description` and a `help` field. Shouldn't the `serve` subcommand property here have the same kind of structure? Then we would have to fix the way it is accessed instead of changing this here.

For the future, these kinds of changes should probably be made in PRs of their own.

Reply:
Thank you. Breaking this out into[ a separate PR ](https://github.com/trufflesuite/truffle/pull/5730)